en/releases/README.md

```diff
@@ -1,8 +1,115 @@
 # Releases
 
-A list of PX4 release notes, they contain a list of the changes that went into each release, explaining the included features, bug fixes, deprecations and updates in detail.
+## Release Notes
+
+List of the changes that went into each release, explaining the included features, bug fixes, deprecations and updates in detail.
 
 * [v1.13](../releases/1.13.md)
 * [v1.12](../releases/1.12.md)
 
 The full archive of releases for the PX4 autopilot project can be found on [GitHub](https://github.com/PX4/PX4-Autopilot/releases)
+
+## Release Versioning
+
+PX4 uses a Semantic versioning (`Major.minor.patch[-beta#]`) system ([Wikipedia article](https://en.wikipedia.org/wiki/Software_versioning)).
+
+- Change in `Major`: Breaking changes
+- Change in `minor`: New non-breaking changes
+- Change in `patch`: All other non-breaking changes
```

Comment:
I challenge this. 

Yes, the version looks like semantic versioning but given we have never bumped the major version in 8 years, we're not really following this. Semantic versioning makes sense when you have a clear API or contract but it's not so easy to define that for PX4. Is it the uORB message definitions? Is it the make interface? Is it the fact that it runs on an STM32 and can be flashed with QGC?

For instance, from v1.13 to v1.14 we're presumably removing the mixers. To me that's a very clear major change and so would definitely warrant a major version bump.

To call it semantic versioning we have to define and agree what we call the API, then based on that we could choose the version number.

Reply:
I agree. The removal of mixer API was absolutely massive for developers. The change to replace RTPS with XRCE-DDS is similarly impactful for developers. 

But I am not sure how else to describe our logic. Perhaps make it clear - we have reserved "major" for some future potential break, and "minor releases" are the significant ones. Or we actually work out what would be "major"

Reply:
> For instance, from v1.13 to v1.14 we're presumably removing the mixers. To me that's a very clear major change and so would definitely warrant a major version bump.

I'm in favour of starting to bump the major version number if we're comfortable doing so going forward (trying much harder to respect some level of semantic versioning) and conveying that intent. From some perspectives removing mixers is a big change, but also keep in mind the vast majority of users are simple multicopters where it's really not much different.

Reply:
> From some perspectives removing mixers is a big change, but also keep in mind the vast majority of users are simple multicopters where it's really not much different.

Right, so we need to decide what our interface/contract is. Is it a user flashing PX4 in QGC and flying a quad, or is it a developer building on top of it.

Reply:
IMO a note for this release is a good approach for now. The reality is that `minor` bumps are actually `major` releases and we have no plan for bumping the `major` number. 

In future it would be good if we agreed criteria and used semver properly. I think the contract for iterating the version should be both users and developers.

Reply:
I added a note on the fact that we will decide during maintainers meeting whether the release will entail major or minor version bump